### ISSUE TITLE
RNG: bad detection of "trng" entropy source

### ISSUE THREAD

**Issue by semenov-vladyslav** [agievich/bee2#20]
`rngHasTRNG` seems to fail to detect presence of `rdseed` instruction which makes `rngReadTRNG` fail. `/proc/cpuinfo` command reports presence of `rdseed`. However, without such detection "trng" entropy source provides entropy successfully passing both `rngTestFIPS1` and `rngTestFIPS2` tests. 

Tested with Intel(R) Core(TM) i9-10885H and linux kernel 5.8.0-50-generic.

**Comment by blackbearman** [comment]
I found that rngCPUID(info, 7); is translated with GCC compiler to instruction
```
__asm__ __volatile__ ("cpuid\n\t" : "=a" (info[0]), "=b" (info[1]), "=c" (info[2]), "=d" (info[3]) : "0" (7));
```
But we need to set both registers EAX=7 and ECX=0 for RDSEED checking by [CPUID](https://en.wikipedia.org/wiki/CPUID#EAX=7,_ECX=0:_Extended_Features):
```
__asm__ __volatile__ ("cpuid" : "=a" (info[0]), "=b" (info[1]), "=c" (info[2]), "=d" (info[3]) : "0" (7), "2" (0));
```
Last instruction works on AMD Ryzen 5 5600H and Linux kernel 5.4.0-131-generic (`/proc/cpuinfo` command reports presence of `rdseed`)

**Comment by blackbearman** [comment]
Problem with rngCPUID was fixed in 8bf552340cf0f7b87d6b1edf044faca6807d032d .
